docs/getting-started/cash-app-pay.md

```diff
@@ -0,0 +1,220 @@
+
+## Step 1: Import the Cash App PayKit Dependency
+
+You can get the the latest version of the SDK from Maven. This is the import definition using Gradle:
+
+```gradle
+implementation "app.cash.paykit:core:1.0.3"
+```
+
+For definitions of other build systems, see [Cash PayKit on Maven Central][cash-on-maven]{:target="_blank"}.
+
+{: .info }
+> Version `v1.0.3` of the SDK size is `12.3 kB`.
+
+## Step 2: Create a Cash App PayKit SDK Instance
+
+To create a new instance of the Cash App PayKit SDK, you must pass the `clientId`. This is a required field. This can be retrieved through the Afterpay object: `Afterpay.environment.payKitId`.
+
+{: .note }
+> Ensure that the Afterpay SDK is configured per the [instructions][configure-afterpay] before attempting to access `Afterpay.environment.payKitId`
+
+
+You should use CashAppPayKitFactory to create an instance of the Cash App PayKit SDK. When doing so, you'll need to specify the environment you will use, Sandbox or Production. The function `createSandbox()` will create an SDK instance in the Sandbox environment.
+
+{: .info }
+> You should use the Sandbox environment during the development phase and Production for your actual production releases.
+
+Creating a sandbox Cash App PayKit SDK instance:
+
+``` kotlin
+val payKit : CashAppPayKit = CashAppPayKitFactory.createSandbox(Afterpay.environment.payKitId)
+```
+
+Creating a production Cash App PayKit SDK instance:
+``` kotlin
+val payKit : CashAppPayKit = CashAppPayKitFactory.create(Afterpay.environment.payKitId)
+```
+
+{: .info }
+**Optional:** We recommend using the [Cash App Sandbox App][sandbox-app]{:target="_blank"} to test the payment flow in the Sandbox environment during your development phase.
+
+## Step 3: Register for State Updates
+
+To receive updates from Pay Kit, you’ll need to implement the `CashAppPayKitListener` interface. The interface exposes a single function, which gets called whenever there’s an internal state change emitted by the SDK:
+
+``` kotlin
+interface CashAppPayKitListener {
+   fun payKitStateDidChange(newState: PayKitState)
+}
+```
+
+You register with the SDK instance you’ve created above:
+
+``` kotlin
+payKit.registerForStateUpdates(this)
+```
+
+You should also use the unregister function when you're done with the SDK:
+
+``` kotlin
+payKit.unregisterFromStateUpdates()
+```
+
+### States
+
+`PayKitState` is a sealed class parameter. We suggest that you use a Kotlin `when` statement on it. Some of these possible states are for information only, but most drive the logic of your integration. The most critical states to handle are in the table below:
+
+| State  | Description |
+|:-------|:------------|
+| `ReadyToAuthorize` | Show a Cash App Pay button in your UI and call `authorizeCustomerRequest()` when it is tapped. |
```

Comment:
I guess I need to correct this on our docs as well. The sentence here is a bit misleading, since it **sounds** like this state will show the button, but in reality what we meant to say here is that it is up to the developer to show the button at this point. So perhaps let's rephrase this as: "You should show the Cash App Pay button ...." .

I'll do the same update on our side.